.github/workflows/FormatCheck.yml

```diff
@@ -1,11 +1,14 @@
 name: "Format Check"
 
 on:
-  push:
-    branches:
-      - 'main'
-    tags: '*'
-  pull_request:
+  pull_request_target:
+    paths: ['**/*.jl']
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
In the latest skeleton I set it up like this: https://github.com/ITensor/ITensorPkgSkeleton.jl/blob/b0db15d60015f22944354b6ee626a80a18f2604f/templates/github/.github/workflows/FormatCheck.yml#L3-L4. What's the difference between the two?